### DISCUSSION TITLE
External database and file system

### DISCUSSION THREAD

**Discussion by maxfirman** [dbt-labs/dbt-server#191] [General]
Currently this project uses local a local sqlite db and file system to persist data about the project state, which means the service loses the project state information following a service restart.

Do you plan to add the option to configure an external database and file system (e.g s3/gcs....), and would this be something that you would accept as a pull request?

**Comment by lostmygithubaccount** [comment]
hi @maxfirman, we appreciate the desire to contribute to `dbt-server`! however, this codebase is under active heavy development and it's unlikely we'll be able to accept any external PRs soon

we are continuing to think through how state is managed in dbt Server, but are unlikely to enable external database/filesystem options
